.gitignore

```diff
@@ -15,3 +15,7 @@
 # Directories
 /.vscode
```

Comment:
Pro tip: I usually do:
```
.*
!.gitignore
```
This ignores all hidden files by default and includes other IDEs too.